view/frontend/templates/js/replacejs.phtml

```diff
@@ -218,6 +218,33 @@ ob_start();
     }());
     ////////////////////////////////////////////////////////////////////////
 
+    ////////////////////////////////////////////////////////////////////////
+    // Wait for an object to be defined and
+    // execute a callback when it becomes available
+    ////////////////////////////////////////////////////////////////////////
+    ! function() {
```

Comment:
no need for wrapping with `! function() { }()`? You would usually want to do this to narrow variable scope but below you are defining global function anyway

Reply:
done